### PR TITLE
test: refactor integration tests

### PR DESCRIPTION
**Pull request by fsamin**
1. Description
 * Changing the way the venom tests are launched to be able to write easily new tests on refactor/authentication branch
 * I try to make them easy to run locally too
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-ui#11084.0 ✘  
* Compile
  * Build UI ✘  
  * Lint Translate ✔ 
  * TU All -   
  * TU model ✘  
  * TU shared ✘  
  * TU store ✘  
  * TU views ✘

**Comment by ovh-cds** [comment]
CDS Report build-ui#11092.0 ✘  
* Compile
  * Build UI ✘  
  * Lint Translate ✔ 
  * TU All -   
  * TU model ✘  
  * TU shared ✘  
  * TU store ✘  
  * TU views ✘

**Comment by ovh-cds** [comment]
CDS Report build-ui#11118.0 ✘  
* Compile
  * Build UI ✘  
  * Lint Translate ✔ 
  * TU All -   
  * TU model ✘  
  * TU shared ✘  
  * TU store ✘  
  * TU views ✘

**Comment by ovh-cds** [comment]
CDS Report build-engine-cli-tools#11119.0 ✘  
* Build minimal
  * cdsctl linux ✔ 
  * check SQL Files number ✔ 
  * Engine Build linux amd64 ✔ 
  * tools/debpacker ✘  
  * tools/os-ansible-inventory ✔ 
  * Worker linux amd64 / 386 ✔

**Comment by ovh-cds** [comment]
CDS Report ut-engine#11224.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report it#11224.0 ✘  
* w-it
  * w-it ✘

**Comment by ovh-cds** [comment]
CDS Report it#11224.0 ✘  
* w-it
  * w-it ✘

**Comment by ovh-cds** [comment]
CDS Report it#11313.0 ✘  
* w-it
  * w-it ✘

**Comment by ovh-cds** [comment]
CDS Report it#11316.0 ✘  
* w-it
  * w-it ✘

**Comment by ovh-cds** [comment]
CDS Report it#11332.0 ✘  
* w-it
  * w-it ✘

**Comment by ovh-cds** [comment]
CDS Report build-ui#11337.0 ✘  
* Compile
  * Build UI ✔ 
  * Lint Translate ✔ 
  * TU All -   
  * TU model ✔ 
  * TU shared ✘  
  * TU store ✔ 
  * TU views ✔

**Comment by ovh-cds** [comment]
CDS Report ut-engine#11343.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report it#11343.0 ✘  
* w-it
  * w-it ✘

**Comment by ovh-cds** [comment]
CDS Report ut-engine#11351.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report it#11351.0 ✘  
* w-it
  * w-it ✘

File: engine/api/workermodel/registration.go [hunk 1]
```diff
@@ -79,10 +78,8 @@ func updateAllToCheckRegistration(db gorp.SqlExecutor) error {
 
 // UpdateSpawnErrorWorkerModel updates worker model error registration
 func UpdateSpawnErrorWorkerModel(db gorp.SqlExecutor, modelID int64, spawnError sdk.SpawnErrorForm) error {
-	// some times when the docker container fails to start, the docker logs is not empty but only contains utf8 null char
-	rs := strings.NewReplacer(string([]byte{0x00}), "", string([]byte{0xbf}), "")
-	spawnError.Error = rs.Replace(spawnError.Error)
-	spawnError.Logs = []byte(rs.Replace(string(spawnError.Logs)))
+	spawnError.Error = sdk.RemoveNotPrintableChar(spawnError.Error)
+	spawnError.Logs = []byte(sdk.RemoveNotPrintableChar(string(spawnError.Logs)))
```
Comment (yesnault) [inline engine/api/workermodel/registration.go]: `0xbf` seems is not removed by `RemoveNotPrintableChar `

https://play.golang.org/p/FNT_J5qgl7J

Comment (fsamin) [reply engine/api/workermodel/registration.go]: osef
this is enough to fix the problem pq utf8 conversion problem we had

finally, this has nothing to deal with the current PR. Don't block it

If you want to fix all weird utf8 unicode problem, fix it in venom, this function is coming from it.

**Comment by ovh-cds** [comment]
CDS Report build-ui#11361.0 ✘  
* Compile
  * Build UI ✔ 
  * Lint Translate ✔ 
  * TU All -   
  * TU model ✘  
  * TU shared ✔ 
  * TU store ✔ 
  * TU views ✔

**Comment by ovh-cds** [comment]
CDS Report it#11366.1 ✘  
* w-it
  * w-it ✘

**Comment by ovh-cds** [comment]
CDS Report it#11379.0 ✘  
* w-it
  * w-it ✘

**Comment by ovh-cds** [comment]
CDS Report ut-engine#11379.0 ✘  
* Stage 1
  * Engine Test with DB ✘

**Comment by ovh-cds** [comment]
CDS Report it#11379.0 ✘  
* w-it
  * w-it ✘

**Pull request merged**